tests/e2e/TestConstants.ts

```diff
@@ -17,6 +17,11 @@ function getBaseUrl(): string {
     return baseUrl.replace(/\/$/, '');
 }
 
+export enum EditorType {
+    THEIA = 'theia',
+    VSCODE = 'code'
```

Comment:
@ScrewTSW : it has come to my mind, that just `code` is too vague short name for VS Code Editor, because it is repo for source code, and every script file is `code` itself.

What if we name it as `vscode` - to avoid confusion in future and to make it equal to `EditorType.VSCODE` constant name?

Reply:
Thanks you for fix.